src/modules/src/esp_deck_flasher.c

```diff
@@ -49,6 +49,105 @@ bool espDeckFlasherCheckVersionAndBoot()
   return true;
 }
 
+static uint32_t sequenceNumber;
+static uint32_t numberOfDataPackets;
+static uint8_t sendBuffer[ESP_MTU + 10 + 16];
+static uint8_t overshoot;
+static uint32_t sendBufferIndex;
+
+bool espDeckFlasherWrite(const uint32_t memAddr, const uint8_t writeLen, const uint8_t *buffer)
+{
+  if (memAddr == 0)
+  {
+    uart2Init(115200);
+    espRomBootloaderInit();
+    if (!espRomBootloaderSync(&sendBuffer[0]))
+    {
+      DEBUG_PRINT("Sync failed\n");
```

Comment:
Do we expect this to ever happen? Can we be more verbose? How will this appear?

`ESPFL: Sync failed` ? 

Perhaps:

```
DEBUG_PRINT("Write failed - cannot sync with bootloader\n");
```